cars/v1/defaults.ini

```diff
@@ -6,4 +6,5 @@ type=car
 base=vanilla
```

Comment:
We also need to change all other configurations; some still reference the default distribution as a config base, e.g. https://github.com/elastic/rally-teams/blob/master/cars/v1/x-pack-security.ini.

Reply:
Oh right, good catch. Updated in https://github.com/elastic/rally-teams/pull/59/commits/0d068afcd302d000d94590a4fabcde184f8ac24c